src/scss/utilities/mixins/_cta.scss

```diff
@@ -83,6 +92,28 @@
       @apply text-gray-700;
     }
   }
-
 }
 
+// This link style is for use
+@mixin cta-dark-vivid-red {
+  @include cta-vivid-red;
+  color: $su-color-vivid-red-dark;
+  @apply no-underline;
+
+  &::after {
+    background-color: $su-color-vivid-red-dark;
+  }
+
+  &.su-link--action,
+  &.su-link--external,
+  &.su-link--download,
+  &.su-link--video,
+  &.su-link--jump,
+  &.su-link--top,
+  &.su-link--internal {
+
+    &::after {
+      background-color: $su-color-vivid-red-dark;
+    }
+  }
```

Comment:
Let's add the comment saying no hover/focus styles are here because it inherits them from the cta-vivid-red mixin.